src/main/java/de/hhu/propra/sharingplatform/dao/ImageService.java

```diff
@@ -0,0 +1,44 @@
+package de.hhu.propra.sharingplatform.dao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.util.MimeType;
+import org.springframework.util.ResourceUtils;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.annotation.Resource;
+import javax.servlet.http.HttpServletRequest;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URL;
+
+@Component
+public class ImageService {
+
+    @Autowired
+    private HttpServletRequest request;
+
+    public void store(MultipartFile file, String filename) {
+        if (file.getContentType().equals("image/png") || file.getContentType().equals("image/jpeg")) {
+            try {
+                File out = ResourceUtils.getFile("src/main/resources/static/images/" + filename);
+                out.createNewFile();
+                FileOutputStream fos = new FileOutputStream(out);
+                fos.write(file.getBytes());
+                fos.close();
+            } catch (IOException exception) {
+                throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR,
```

Comment:
We can make a better response if we check the file size first. To big files should throw a different error

Reply:
Good idea